src/components/Avatar.tsx

```diff
@@ -9,11 +10,26 @@ type AvatarProps = {
 
 // name = friendlyName ?? identity
 const Avatar: React.FC<AvatarProps> = ({ name, size }) => {
-	if (name.startsWith("whatsapp:") || name.startsWith("sms:") || name.startsWith("+")) {
-		return <PasteAvatar size={size ?? "sizeIcon70"} variant="user" name={name} icon={UserIcon} />
-	}
-	return <PasteAvatar size={size ?? "sizeIcon70"} variant="user" name={name} />
-	// use src for specified avatar images - once we have them!
```

Comment:
All of these formatting changes are a result of `prettier` not allowing me to even run the app.